### PR TITLE
Add some accessory methods to styleUtils and export some service methods for testcafe (close #314)

### LINKED ISSUE #314: Add some accessory methods in styleUtils and export some service methods for getting rid of jQuery in testcafe
https://github.com/DevExpress/testcafe/issues/56

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @inikulin @churkin

### CODE REVIEW

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

File: src/client/index.js [hunk 1]
```diff
@@ -68,7 +71,12 @@ class Hammerhead {
             event:    eventUtils,
             position: positionUtils,
             style:    styleUtils,
-            types:    typeUtils
+            types:    typeUtils,
+            service:  {
+                isJQueryObj: isJQueryObj,
+                extend:      extend,
+                trim:        trim
+            }
```
Comment (churkin) [inline src/client/index.js]: Maybe we should add methods to the utils root instead of creating a new `service` entity?

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
\r

**Comment by inikulin** [comment]
Please, review without me

File: src/client/index.js (hunk 1 above)
Comment (AlexanderMoskovkin) [inline src/client/index.js]: I'm agree with this. We have separated modules for this utils, so we should add the methods to the utils root.

**Comment by helen-dikareva** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by churkin** [comment]
lgtm

**Pull request merged**